js/ui/drawer/ui.drawer.js

```diff
@@ -250,7 +251,7 @@ const Drawer = Widget.inherit({
 
         const contentTemplateOption = this.option("contentTemplate"),
             contentTemplate = this._getTemplate(contentTemplateOption),
-            transclude = this._getAnonymousTemplateName() === contentTemplateOption;
+            transclude = this._templateManager.anonymousTemplateName === contentTemplateOption;
```

Comment:
Can we add the isTranscludedTemplate(templateOptionValue) method to _templateManager?
Similar expression:
transclude = this._templateManager.anonymousTemplateName === contentTemplateOption
exists in 5 places.

Reply:
I think no because this method will show only that template is an anonymous template. It is not a greate logic that should be written in the TemplateManager API and this method will be created on each instance creation, this is bad. Current API is little and flexible.